README.md

```diff
@@ -80,6 +80,147 @@ async nuxtServerInit({ commit }, { $sentry }) {
 }
 ```
 
+## Lazy Loading (on the client)
```

Comment:
Maybe this section should be after the `Options` section? Options are probably the first thing (after "setup") that people should look at and it refers to more details "lazy loading" section.

Reply:
To me options is more like a reference, while Lazy Loading is more part of the docs explaning functionality. Because of that I used this order.

Eg if we would have a vuepress website for the docs then Lazy Loading would be under Guide but Options would be under API. Agree that the lazy options could be moved down maybe but thought it would be better to keep all Lazy loading stuff together.

Reply:
OK. Do what you prefer.

Reply:
Have added a Configure section after Setup with a shortcut to Options, that way its still the first thing after setup in the readme :)